libs/sdk-backend-mockingbird/src/recordedBackend/tests/execution.test.ts

```diff
@@ -26,12 +26,6 @@ describe("recordedDataView", () => {
         expect(dv.result).toBeDefined();
     });
 
-    it("should load result with uriRefs by default", () => {
-        const dv = recordedDataView(ReferenceRecordings.Scenarios.BarChart.SingleMeasureWithViewBy);
-
-        assertExpectedRefs(dv.result, isUriRef);
```

Comment:
Should we change the default to idRefs, and just update the tests (assert isIdRef)?

Reply:
fixed